src/components/entity/ha-entity-picker.ts

```diff
@@ -57,6 +57,22 @@ export type HaEntityPickerEntityFilterFunc = (entity: HassEntity) => boolean;
 
 const CREATE_ID = "___create-new-entity___";
 
+const DOMAIN_STYLE = styleMap({
```

Comment:
still don't understand why it has to be a `styleMap`, it is not dynamic?

Reply:
For maintenance. Having a string make it hard to read and maintain. It's now outside of the component so it will be computed only one time.